src/server/render.js

```diff
@@ -27,10 +23,14 @@ export default (request, reply) => {
         </App>
       );
 
-      const stateJSON = asciiJSON.stringify(initialState).replace(/<\//g, '<\\/');
-      const webserver = process.env.NODE_ENV === 'production' ? '' : '//localhost:8080';
+      const basePath = process.env.NODE_ENV === 'production' ? '' : '//localhost:8080';
 
-      reply(markup(reactString, stateJSON, webserver, config));
+      reply.view('index', {
+        reactMarkup: reactString,
+        initialState: JSON.stringify(initialState),
+        javascripts: [`${basePath}/dist/client.js`],
+        stylesheets: process.env.NODE_ENV === 'production' ? ['/dist/styles.css'] : ''
+      });
```

Comment:
@pgoetze let's think about how we handle all the 'markup' varibles, especially when combined with over-rides / additions from the user's theme